docs/installation.rst

```diff
@@ -34,16 +34,8 @@ Dependencies
 Python Packages
 ^^^^^^^^^^^^^^^
 
-The project has a few common Python package dependencies. The runtime
-dependencies are:
-
-.. include:: ../requirements.txt
-   :literal:
-
-The build time dependencies (also required for development) are:
-
-.. include:: ../requirements-dev.txt
-   :literal:
+The project has a few common Python package dependencies. These can be seen in
+``setup.py``.
```

Comment:
```suggestion
The project has a few common Python package dependencies. These can be seen in
``setup.py`` and ``pyproject.toml``.
```

The old list is incorrect, things like `flake8` are not a requirement, and coverage was not a build time requirement. Nowadays, listing requirements in docs is not nearly as common.